### ISSUE TITLE
code improvement

### ISSUE THREAD

**Issue by hvgazula** [hassonlab/247-encoding#34]
Replace [this](https://github.com/hassonlab/247-encoding/blob/main/code/tfsenc_main.py#L231-L239) piece of code with

```
prod_train = prod_X, prod_Y, fold_cat_prod
comp_train = comp_X, comp_Y, fold_cat_comp

if args.model_mod == 'pc-flip':
    prod_test, comp_test = comp_train, prod_train
else:
    prod_test, comp_test = prod_train, comp_train

prod_results = run_regression(args, *prod_train, *prod_test, fold_num)
comp_results = run_regression(args, *comp_train, *comp_test, fold_num)
```

**Comment by hvgazula** [comment]
@zkokaja what do you think?

**Comment by hvgazula** [comment]
I guess you cannot have an unnamed argument after *args, either move `fold_num` to the beginning or add a default value.

**Comment by VeritasJoker** [comment]
Ok, thanks Harsha and Zaid! I will modify those and create a new pull request.

**Comment by zkokaja** [comment]
Closed pull pull request #35
